ninja-core/src/site/markdown/developer/getting_started.md

```diff
@@ -33,6 +33,19 @@ send us a pull request to become a happy member of the Ninja family :)
 
 ### Code style
 
+- **Don't use null**
```

Comment:
@jjlauer Does that makes sense to you? I added some guidelines...

Reply:
These are good.  I haven't personally switched to AssertJ myself, but I checked them out and they look interesting.  I am happy to give them a shot.